service/src/main/java/bio/terra/workspace/service/resource/controlled/flight/clone/azure/database/DumpAzureDatabaseStep.java

```diff
@@ -124,7 +124,7 @@ public StepResult doStep(FlightContext context) throws InterruptedException, Ret
     this.azureDatabaseUtilsRunner.pgDumpDatabase(
         sourceAzureContext,
         sourceDatabase.getWorkspaceId(),
-        "dump-db-" + this.sourceDatabase.getResourceId(),
+        "dump-db-%s-%s".formatted(sourceDatabase.getDatabaseName(), destinationContainer.getResourceId()),
```

Comment:
I have updated this to also include the source database being cloned to avoid another name collision when more databases in a workspace (in future) are cloned using this same mechanism.